src/asInput/index.jsx

```diff
@@ -148,16 +151,17 @@ const asInput = (WrappedComponent, inputType = undefined, labelFirst = true) =>
     }
 
     getLabel() {
+      const { label } = this.props;
       return (
         // eslint-disable-next-line jsx-a11y/label-has-for
         <label
           id={`label-${this.state.id}`}
           htmlFor={this.state.id}
-          className={[classNames({
+          className={classNames({
             [styles['form-check-label']]: this.isGroupedInput(),
-          })]}
+          })}
         >
-          {this.props.label}
+          {label}
```

Comment:
nit: label is only used once, could pull it from `this.props` here instead of pulling it out above